storybook/pages/JoinCommunityViewPage.qml

```diff
@@ -0,0 +1,211 @@
+            ListElement { name: "welcome"; selected: false; notificationsCount: 0; hasUnreadMessages: false}
+            ListElement { name: "general"; selected: false; notificationsCount: 0; hasUnreadMessages: true}
+            ListElement { name: "design"; selected: true; notificationsCount: 3; hasUnreadMessages: true}
+            ListElement { name: "random"; selected: false; notificationsCount: 0; hasUnreadMessages: false}
+            ListElement { name: "vip"; selected: false; notificationsCount: 0; hasUnreadMessages: true}
+        }
+        readonly property ListModel model2:  ListModel {
+            ListElement { name: "general"; selected: false; notificationsCount: 3; hasUnreadMessages: false}
+            ListElement { name: "blockchain"; selected: true; notificationsCount: 3; hasUnreadMessages: true}
+            ListElement { name: "faq"; selected: false; notificationsCount: 0; hasUnreadMessages: false}
+        }
+        property var messagesModel: ListModel {
+            ListElement {
+                timestamp: 1656937930
+                senderDisplayName: "simon"
+                contentType: StatusMessage.ContentType.Text
+                message:  "Hello, this is awesome! Feels like decentralized Discord!"
+                isContact: true
+                trustIndicator: StatusContactVerificationIcons.TrustedType.Verified
+                colorId: 4
+            }
+            ListElement {
+                timestamp: 1657937930
+                senderDisplayName: "Mark Cuban"
+                contentType: StatusMessage.ContentType.Text
+                message: "I know a lot of you really seem to get off or be validated by arguing with strangers online but please know it's a complete waste of your time and energy"
+                isContact: false
+                trustIndicator: StatusContactVerificationIcons.TrustedType.Untrustworthy
+                colorId: 2
+            }
+        }        
+    }
+
+    Logs { id: logs }
+
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        Rectangle {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+            color: Theme.palette.statusAppLayout.rightPanelBackgroundColor
+            clip: true
+
+            JoinCommunityView{
+                anchors.fill: parent
+                anchors.margins: 50
+
+                // General properties:
+                name: d.name
+                communityDesc: d.communityDesc
+                color: d.color
+                channelName: d.channelName
+                channelDesc: d.channelDesc
+                joinCommunity: d.joinCommunity
+
+                // Blur background properties:
+                membersCount: d.membersCount
+                image: d.image                
+                amISectionAdmin: d.amISectionAdmin
+                openCreateChat: false
+                communityItemsModel: d.communityItemsModel
```

Comment:
You are right but I decided to keep it now only in `storybook` and take the decision when it is integrated since maybe we can decide to load some "real" model instead. WDYT?

Reply:
IMO there is no chance to have access to real data at this stage. It would be a big problem bc users could easily change source code and "unblock" access to resources protected by our privileges system. Protocol should really protect access, it's not only matter of UI. We could as @John-44 to clarify that.

Reply:
Well, at this point, what I'm thinking of as "real" data is just having a model that contains the names of the channels the community has, nothing else, and maybe... the color of the community. Nothing that could be vulnerable. Only something that makes the UI more interactive.